.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Type of change (place an x in the [ ] that applies)
+
+- [ ] New application
+- [ ] Bug fix (non-breaking change which fixes an issue) 
+- [ ] New feature (non-breaking change which adds functionality) 
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
+- [ ] Documentation update
```

Comment:
```suggestion
- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [ ] Documentation
```

Since the PRs that will be opened are not dependencies for existing projects, the idea of a breaking change doesn't apply so cleanly. Let's hold off on that category until we have a decided path forward for template versioning, then we can circle back and adjust.